js/src/forum/components/ErasureRequestsDropdown.js

```diff
@@ -0,0 +1,47 @@
+/*
+ *
+ *  This file is part of fof/erasurerequests.
+ *
+ *  Copyright (c) 2019 FriendsOfFlarum.
+ *
+ *  For the full copyright and license information, please view the LICENSE.md
+ *  file that was distributed with this source code.
+ *
+ */
```

Comment:
This be `blomstra/flarum-gdpr`, etc

I can look at setting up StyleCI to handle the file headers